src/routes/TabNavigator.js

```diff
@@ -46,19 +46,32 @@ const BottomTabNavigator = () => {
           backgroundColor: '#FEF8E3',
         },
       }}>
-      <>
-        <Tab.Screen name='Home' component={ResourcesTabNavigator} />
-        <Tab.Screen name='Hotlines' component={SheltersHotlinesTabNavigator} />
-        {state.token && (
+      {!state.token ? (
```

Comment:
so we do not need this logic anymore, we just create a new task to make a blocked sos screen as designers have, but you can may be merge now.